builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -58,7 +58,7 @@ def loadServiceConfigurationData() {
             updateConfigValue("{jazz_admin}", config_loader.JAZZ.ADMIN);
             updateConfigValue("{jazz_admin_creds}", config_loader.JAZZ.PASSWD);
  
-            sh "sed -i -- 's/{conf-region}/${region}/g' ./config/global-config.json"
+            sh "sed -i -- 's/{cf-region}/us-east-1/g' ./config/global-config.json"
```

Comment:
add a comment on why was this done. Add the same in metrics api file too

Reply:
updated.